src/frame.rs

```diff
@@ -34,8 +34,7 @@ use crate::Result;
 pub(crate) const MAX_STREAM_SIZE: u64 = 1 << 62;
 
 pub(crate) const MAX_CRYPTO_OVERHEAD: usize = 8;
-pub(crate) const MAX_STREAM_OVERHEAD: usize = 12;
-pub(crate) const MIN_STREAM_OVERHEAD: usize = 5;
+pub(crate) const MAX_STREAM_OVERHEAD: usize = 19;
```

Comment:
Give some more explanations on the modification (like what fields cause the MAX_STREAM_OVERHEAD to change and what does the 19 come from)

Reply:
Sure. I will add the comments from the official repository here.